commands/openers.go

```diff
@@ -55,7 +59,12 @@ var projectLocalOpenCmd = &console.Command{
 		if !pidFile.IsRunning() {
 			return console.Exit("Local web server is down.", 1)
 		}
-		abstractOpenCmd(fmt.Sprintf("%s://127.0.0.1:%d", pidFile.Scheme, pidFile.Port))
+		url := fmt.Sprintf("%s://127.0.0.1:%d", pidFile.Scheme, pidFile.Port)
+		webPath := c.String("webPath")
+		if webPath != "" {
```

Comment:
A minor thing: The recommended way to write this in Go is

```go
if webPath := c.String("webPath"); webPath != "" { … }
```

cf. https://go.dev/doc/effective_go#if.

Reply:
`c.String("webPath")` should always return a string (worst case an empty one)
so you can probably simplify even more and remove the if, like so:
`abstractOpenCmd(fmt.Sprintf("%s://127.0.0.1:%d/%s", pidFile.Scheme, pidFile.Port, strings.TrimLeft(c.String("webPath"))))`

Reply:
This will leave us with a trailing slash if the `webPath` flag is not set, but that's not an issue.